### ISSUE TITLE
Most Worst and Annoying Feature, Reconnect pop up window all the time

### ISSUE THREAD

**Issue by sam1730** [appirio-tech/arena-web#725]
Hi,

I found very bad response time issue on this arena-web, to pop-up window that you are not connected to the Internet, I think the highest security risk prevention to the website have made it to respond very slow and very buggy, All other websites are working fine but arena.topcoder.com says that the user is not connected to the internet. Please fix this bug seems to be very irritating while giving tests and practicing problems.

**Comment by bbatjargal** [comment]
Yep, I have the same problem as you. It says all the time that disconnected and reconnect. Also, it is quite running slow and displaying that "Please wait while the requested problem is loaded. This may take several minutes ...".
